docs/reference/data-frames/apis/stop-transform.asciidoc

```diff
@@ -68,6 +71,11 @@ If this parameter is `false`, the request returns a `404` status code when there
 are no matches or only partial matches.
 --
 
+`force`::
+  (Optional, boolean) If `true`, resets the state of a failed job back to 
+  started. It is required to do in order to delete or restart the job (after 
```

Comment:
This seems unexpected.  Does "stopping" the transform forcefully really put it into "started" state?

Reply:
I would have expected the description for this parameter to be similar to the force parameter on the close job API: https://www.elastic.co/guide/en/elasticsearch/reference/master/ml-close-job.html

Reply:
@lcawl is correct

Reply:
Sorry, my bad. I corrected it.